exercises/mongo-shell/intermediate-read.md

```diff
@@ -108,6 +119,58 @@ Use the `$nin` query operator (**not in** operator) to match **none** of the val
 
 ---
 
+## Logical Operators
+
+### The "$or" operator
+This operator **OR**s the filters stated in a query and returns documents where atleast 1 of the filters is true. 
+
+**Exercise 1** :computer: 
+
+Write a query to list movies whose `imdb.rating` is greater than 8.5 or `metacritic` rating is greater than 85.
+
+:arrow_right: Note that the `$or` operator takes an **array** of values. 
+
+**query:**
+```javascript
+db.movieDetails.find({"$or": [
+    {"imdb.rating": {"$gt": 8.5}}, 
+    {"metacritic": {"$gt": 85}}]}, 
+        {"_id": 0, "title": 1, "imdb.rating": 1, "metacritic": 1})
+```
+---
+
+### The "$and" operator
+
+As you already know, MongoDB implicitly **AND**s the filters when separated by `,` in the query. 
+
+**Exercise 2** :computer: 
+
+Run the following query:
```

Comment:
explain some objective or 'why' behind running the following query.  I know you go into detail when talking about the output below, but 'run the following query' is asking the student to blindly follow you.  

```suggestion
To cover this next point we need to generate some output first.  Run the query below to get the output we need. 

➡️ Can you explain the different operators and parameters in this query without seeing the output?
```